### PR TITLE
Merge security setup into new section

### PR DESCRIPTION
**Pull request by jgordini**
added a basic security setup section and move security info into it. 
moved floating ip into networking.

### CODE REVIEW

File: docs/uab_cloud/security_setup_basic.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Basic Security Setup
+
+Security Groups can and should be added as needed.
```
Comment (wwarriner) [inline docs/uab_cloud/security_setup_basic.md:3]: Can we add a little bit more to this intro? I'd like to maintain the intent of these docs to be a walkthrough for mostly-novice users. Something like the following:

```Using SSH with your instances requires following the steps on this page. At a minimum, an SSH security group and at least one keypair must be created. Other security groups can and should be added as needed.```

File: mkdocs.yml [hunk 2]
```diff
@@ -65,5 +65,6 @@ nav:
   - UAB Cloud:
     - Introduction: 'uab_cloud/introduction.md'
     - 'uab_cloud/network_setup_basic.md'
+    - 'uab_cloud/security_setup_basic.md'
```
Comment (wwarriner) [inline mkdocs.yml:68]: Appreciate keeping things in order!

**Review by wwarriner — CHANGES_REQUESTED** [review]

**Comment by wwarriner** [comment]
Good stuff, appreciate your taking time to make contributions! I like this change, just a tiny bit to make sure we keep the "walkthrough" flavor for these pages.

File: docs/uab_cloud/security_setup_basic.md (hunk 1 above)
Comment (jgordini) [reply docs/uab_cloud/security_setup_basic.md]: Updated the security intro. Do I need to resubmit the merge request?

Comment (jgordini) [reply docs/uab_cloud/security_setup_basic.md]: looks like it includes the new commit. still working on my git knowledge.

File: mkdocs.yml (hunk 2 above)
Comment (jgordini) [reply mkdocs.yml:68]: No problem. :)

**Review by wwarriner — APPROVED** [review]

**Pull request merged**